dot-atom/keymap.cson

```diff
@@ -21,5 +22,6 @@
 # * https://atom.io/docs/latest/behind-atom-keymaps-in-depth
 #
 # This file uses CoffeeScript Object Notation (CSON).
-# If you are unfamiliar with CSON, you can read more about it here:
-# https://github.com/bevry/cson#what-is-cson
+# If you are unfamiliar with CSON, you can read more about it in the 
+# Atom Flight Manual:
+# https://atom.io/docs/latest/using-atom-basic-customization#cson
```

Comment:
Btw @izuzak, I changed this link to the CSON Primer I added in https://github.com/atom/docs/pull/100 – do you think that's okay? There is a footnote in that section to the original link here, and it expands upon the source CSON docs quite a bit.